api/policies/isMod.js

```diff
@@ -1,23 +0,0 @@
-var http = require("http");
```

Comment:
Don't delete this entirely. But now that we have user.isMod defined, we should keep this check and use it the same way as sessionAuth.

Basically, if isMod is not true, then we return 403. Then we can remove all the if (!user.isMod) from all the places that mods are only meant to be accessing.